.github/workflows/build.yml

```diff
@@ -370,6 +370,7 @@ jobs:
       - name: "[macOS] Sign, Notarize, and Staple Package"
         if: runner.os == 'macOS' && env.MACOS_CODESIGN_OPENSSL_PASSWORD != null && env.MACOS_CODESIGN_CERTIFICATE_PASSWORD != null && env.APPLE_APP_SPECIFIC_PASSWORD != null
         run: packaging/macos/sign_notarize_staple.sh build/*.dmg
+        timeout-minutes: 5
```

Comment:
not sure, but that might a little conservative. Apple might take much longer. I guess we can keep it low for now and increase it if it turns out to be a problem. I think the better solution would be to terminate the script early if we received a status that is known to be final (such as "invalid").

Reply:
I looked up how long it took last time it worked, it was ~3min. But yeah, it might take longer. Let's keep it that low for testing, and whoever actually fixes the notarization failure can set it to some reasonable value.